src/generator.cs

```diff
@@ -2102,7 +2104,7 @@ public void Go ()
 			marshal_types.Add (new MarshalType (TypeManager.ABRecord, create: "ABRecord.FromHandle("));
 		}
 		if (Frameworks.HaveCoreVideo)
-			marshal_types.Add (TypeManager.CVPixelBuffer);
+			marshal_types.Add (new MarshalType (TypeManager.CVPixelBuffer, create: "Runtime.GetINativeObject<CVPixelBuffer> (", closingCreate: ", false)")); // owns `false` like ptr ctor https://github.com/xamarin/xamarin-macios/blob/6f68ab6f79c5f1d96d2cbb1e697330623164e46d/src/CoreVideo/CVBuffer.cs#L74-L90
```

Comment:
put comments on the previous line (it's too easy to miss at the end of a long line)
add `{}` when needed